arelle/plugin/validate/DBA/rules/tr.py

```diff
@@ -192,6 +194,41 @@ def rule_tr09(
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_tr11(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    DBA.TR11: InlineXBRL must not contain references to external images. Only Base64 encoded content is accepted.
+    """
+    imagesToCheck = set()
+    _xhtmlNs = "{{{}}}".format(xhtml)
+    _xhtmlNsLen = len(_xhtmlNs)
+    modelXbrl = val.modelXbrl
+    modelDocument = modelXbrl.modelDocument
+    if modelDocument is not None and modelDocument.type in (ModelDocument.Type.INLINEXBRL, ModelDocument.Type.INLINEXBRLDOCUMENTSET):
+        for ixdsHtmlRootElt in modelXbrl.ixdsHtmlElements:
+            for elt in ixdsHtmlRootElt.iter():
+                if isinstance(elt, (_ElementTree, _Comment, _ProcessingInstruction)):
+                    continue # comment or other non-parsed element
+                eltTag = elt.tag
+                if eltTag.startswith(_xhtmlNs):
+                    eltTag = eltTag[_xhtmlNsLen:]
+                    if eltTag == "img":
+                        imagesToCheck.add(elt.get("src","").strip())
+    if len(imagesToCheck) > 0:
+        return errorOnForbiddenImage(
+            images = imagesToCheck,
+            code="DBA.TR11",
+            message=("InlineXBRL must not contain references to external images. Additionaly only Base64 encoded content is accepted."),
+        )
+    return []
```

Comment:
You should yield from the generator instead of returning it:
```suggestion
        yield from errorOnForbiddenImage(
            images = imagesToCheck,
            code="DBA.TR11",
            message=("InlineXBRL must not contain references to external images. Additionaly only Base64 encoded content is accepted."),
        )
```